tools/nni_cmd/launcher_utils.py

```diff
@@ -130,26 +147,31 @@ def parse_assessor_content(experiment_config):
             if not os.path.exists(os.path.join(
                     experiment_config['assessor']['codeDir'],
                     experiment_config['assessor']['classFileName'])):
-                raise ValueError('Assessor file directory is not valid!')
+                print_error('Assessor file directory is not valid!')
+                exit(1)
         else:
-            raise ValueError('Assessor format is not valid!')
+            print_error('Assessor format is not valid!')
+            exit(1)
 
 def validate_annotation_content(experiment_config):
     '''Valid whether useAnnotation and searchSpacePath is coexist'''
     if experiment_config.get('useAnnotation'):
         if experiment_config.get('searchSpacePath'):
-            raise Exception('If you set useAnnotation=true, please leave searchSpacePath empty')
+            print_error('If you set useAnnotation=true, please leave searchSpacePath empty')
+            exit(1)
     else:
         # validate searchSpaceFile
         if experiment_config['tuner'].get('tunerName') and experiment_config['tuner'].get('optimizationMode'):
             if experiment_config.get('searchSpacePath') is None:
-                raise Exception('Please set searchSpace!')
+                print_error('Please set searchSpace!')
```

Comment:
'Please set searchSpace!' , should the searchSpace be searchSpacePath?

Reply:
updated to searchSpacePath